### ISSUE TITLE
Bug: Field name cannot contain dot when using ActionsInTabs

### ISSUE THREAD

**Issue by sirikkoster** [eminiarts/nova-tabs#94]
- Laravel Version: 7.3.16
- Nova Version: 3.4.0
- PHP Version: 7.2.29

### Description:
If i use the trait ActionsInTabs and there are dot's in the field names the value will not be shown

### Steps To Reproduce:

Shows nothing
```php
Text::make('my.Id')->displayUsing(function(){
     return $this->id;
})
```

Shows the value
```php
Text::make('my_Id')->displayUsing(function(){
     return $this->id;
})
```

**Comment by RVxLab** [comment]
Is this still an issue at this point? If so, can you provide me with a way to reliably reproduce this? Thanks!

**Comment by RVxLab** [comment]
With the latest changes in master (to be released in 1.4.0) this should no longer be an issue.
